docs/src/main/paradox/errorhandling.md

```diff
@@ -9,24 +9,16 @@ When the last retry fails, source stage will be failed with an exception.
 
 Retry handling in case of producer is built-in into Kafka. In case of failure when sending a message, an exception will be thrown, which should fail the stream. 
 
-## Restarting the stream
+## Restarting the stream with a backoff stage
 
-Typical approach is to run the stream inside an actor. When there's an exception, this actor should be stopped and a new one should be created.
-Stopping the actor on stream failure:
+Akka streams [provides graph stages](https://doc.akka.io/docs/akka/2.5.5/java/stream/stream-error.html#delayed-restarts-with-a-backoff-stage)
```

Comment:
I think you can use `@extref[provides graph stages](akka-docs:stream/stream-error.html#delayed-restarts-with-a-backoff-stage)` to avoid hard-coding the `2.5.5` here

Reply:
Yup, that worked nicely.